src/scss/layout/_footer.scss

```diff
@@ -48,3 +52,9 @@ $two-column: 45em;
 
   font-size: var(--xsmall);
 }
+
+.oddnews-signup {
+  background-color: var(--clip-bg, var(--callout));
```

Comment:
@SondraE @sanajaved7 do we want this background? I thought originally it had one since the default mailchimp styles included a bg color.

Reply:
@stacyk we took out the Mailchimp styles so there wasn't a background color after that and in standup @SondraE mentioned it looked confusing to have the two forms next two each other and we wanted to use a background color to distinguish the forms from each other.

In Slack we looked at this callout color and then also the `callout-special-dark` color and decided to go with this one. But it might be good that now that we have the deploy preview, @SondraE you can take a look at the footer [here](https://deploy-preview-353--oddleventy.netlify.app/) and see if this color looks good still.

Reply:
@sanajaved7 @stacyk Took a look. I still like how it looks having this background color for this form.